sentry_sdk/integrations/django/__init__.py

```diff
@@ -358,9 +363,33 @@ def record_sql(sql, params, cursor=None):
         except Exception:
             pass
 
+    span = None
+
     if real_sql:
         with capture_internal_exceptions():
             hub.add_breadcrumb(message=real_sql, category="query")
+            span = hub.start_span(op="sql.query", description=real_sql)
+
+    if span is None:
+        yield
+    else:
+        try:
+            yield
+        finally:
+            span.set_tag("status", sys.exc_info()[1] is None)
+            span.finish()
+
+
+@contextlib.contextmanager
+def record_many_sql(sql, param_list, cursor):
+    ctxs = [record_sql(sql, params, cursor).__enter__() for params in param_list]
```

Comment:
It seems to be that a span here should represent the roundtrip to the server. If that had multiple sql statements, we should still have a single Span.

Reply:
I convert multiple statements to a single semicolon-joined one now. One span everytime.